src/core/algorithm/distributionToCred.js

```diff
@@ -59,51 +51,48 @@ export function distributionToCred(
   nodeOrder: $ReadOnlyArray<NodeAddressT>,
   scoringNodePrefixes: $ReadOnlyArray<NodeAddressT>
 ): TimelineCredScores {
-  if (ds.length === 0) {
-    return {intervals: [], intervalCredScores: []};
-  }
   const scoringNodeIndices = [];
   for (let i = 0; i < nodeOrder.length; i++) {
     const addr = nodeOrder[i];
     if (scoringNodePrefixes.some((x) => NodeAddress.hasPrefix(addr, x))) {
       scoringNodeIndices.push(i);
     }
   }
-  const intervals = ds.map((x) => x.interval);
-  const intervalCredScores = ds.map(({distribution, intervalWeight}) => {
+  return ds.map(({interval, distribution, intervalWeight}) => {
     const intervalTotalScore = sum(
       scoringNodeIndices.map((x) => distribution[x])
     );
 
     const intervalNormalizer =
       intervalTotalScore === 0 ? 0 : intervalWeight / intervalTotalScore;
     const cred = distribution.map((x) => x * intervalNormalizer);
-    return cred;
+    return {interval, cred};
   });
-  return {intervalCredScores, intervals};
 }
 
-const COMPAT_INFO = {type: "sourcecred/timelineCredScores", version: "0.1.0"};
+const COMPAT_INFO = {type: "sourcecred/timelineCredScores", version: "0.2.0"};
```

Comment:
This serialization format seems to be unused other than it's unit tests.
Prefer a preceding PR to remove it, rather than updating it.